[ARO] az aro: Fix table output

m1kola opened the pull request:
**Description**

Fixes a bug in `az aro {command} --output table` (`az aro list --output table`, for example) when the comamnd fails due to not taking into account potential `None` values in data in some edge cases table output currently fails. For example:

* When there are no worker profiles (cluster only has masters or we failed to get data from API server)
* When a worker profile does not have count (machine set was scaled to 0)

**Testing Guide**  

Create an ARO cluster, scale all machine sets to 0 and try to run `az aro list --output table`.

**History Notes**  
<!--If your PR is not customer-facing, use {Component Name} in the PR title. Otherwise, use [Component Name] to allow our pipeline to add the title as a history note. If you need multiple history notes or would like to overwrite the note from the PR title, please fill in the following templates.-->

[ARO] az aro: Fix table output

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

yonzhan commented:
add to S169

haroldrandom commented:
Could you please add \_\_init\_\_.py under `tests/` and its child directory `latest/`?

m1kola commented:
Good catch! Added `__init__.py` into these dirs. But I think we also need to update request recordings. I'll look into how to do it now.

haroldrandom commented:
> Good catch! Added `__init__.py` into these dirs. But I think we also need to update request recordings. I'll look into how to do it now.

One reason why CI failed is azure-mgmt-network has bumped to a newer version, you could re-recordin test to resolve this.
But still need you fix the problem for your modifcation if any.

m1kola commented:
@jiasli @haroldrandom please take a look. I rebased on top of `dev` branch which now has `__init__.py` files for tests and new recordings and added some extra test for table output.
